pkg/cableengine/cableengine.go

```diff
@@ -117,3 +119,7 @@ func (i *engine) RemoveCable(endpoint types.SubmarinerEndpoint) error {
 	klog.Infof("Successfully removed Endpoint cable %q", endpoint.Spec.CableName)
 	return nil
 }
+
+func (i *engine) ListCableConnections() (*[]cable.Connection, error) {
+	return nil, fmt.Errorf("Not implemented yet")
```

Comment:
Will this also require a new method on the `cable.Driver` interface?I see there's a `GetActiveConnections` method but that takes a clusterID.

Reply:
Of course, I was planning to make that happen during the real implementation, This was just defining the high level API and structures (which will also be used for reporting)

I was considering the driver part as part of the implementation that we can discuss later.

I can add it here too, no harm :-)